packages/browser/src/index.test.ts

```diff
@@ -86,346 +66,39 @@ jest.mock('jose', () => ({
-      await logtoClient.signOut(postSignOutRedirectUri);
-      const encodedRedirectUri = encodeURIComponent(postSignOutRedirectUri);
-
-      expect(window.location.toString()).toEqual(
-        `${endSessionEndpoint}?id_token_hint=id_token_value&post_logout_redirect_uri=${encodedRedirectUri}`
-      );
-    });
-
-    it('should not block sign out flow even if token revocation is failed', async () => {
-      const logtoClient = new LogtoClient({ endpoint, appId }, failingRequester);
-
-      await expect(logtoClient.signOut()).resolves.not.toThrow();
-      expect(failingRequester).toBeCalledTimes(1);
-      expect(localStorage.getItem(idTokenStorageKey)).toBeNull();
-      expect(localStorage.getItem(refreshTokenStorageKey)).toBeNull();
-      expect(window.location.toString()).toEqual(
-        `${endSessionEndpoint}?id_token_hint=id_token_value`
-      );
-    });
-  });
-
-  describe('getAccessToken', () => {
-    it('should throw if idToken is empty', async () => {
-      localStorage.removeItem(idTokenStorageKey);
-      localStorage.setItem(refreshTokenStorageKey, 'refresh_token_value');
-      const logtoClient = new LogtoClient({ endpoint, appId }, requester);
-
-      await expect(logtoClient.getAccessToken()).rejects.toMatchError(
-        new LogtoClientError('not_authenticated')
-      );
-    });
-
-    it('should throw if refresh token is empty', async () => {
-      localStorage.setItem(idTokenStorageKey, 'id_token_value');
-      localStorage.removeItem(refreshTokenStorageKey);
-      const logtoClient = new LogtoClient({ endpoint, appId }, requester);
-
-      await expect(logtoClient.getAccessToken()).rejects.toMatchError(
-        new LogtoClientError('not_authenticated')
-      );
-    });
-
-    it('should return access token by valid refresh token', async () => {
-      // eslint-disable-next-line @silverhand/fp/no-let
-      let count = 0;
-
-      requester.mockClear().mockImplementation(async () => {
-        // eslint-disable-next-line @silverhand/fp/no-mutation
-        count += 1;
-        await new Promise((resolve) => {
-          setTimeout(resolve, 0);
-        });
-
-        return {
-          accessToken: count === 1 ? 'access_token_value' : 'nope',
-          refreshToken: count === 1 ? 'new_refresh_token_value' : 'nope',
-          expiresIn: 3600,
-        };
-      });
-      localStorage.setItem(idTokenStorageKey, 'id_token_value');
-      localStorage.setItem(refreshTokenStorageKey, 'refresh_token_value');
-
-      const logtoClient = new LogtoClient({ endpoint, appId }, requester);
-      const [accessToken_1, accessToken_2] = await Promise.all([
-        logtoClient.getAccessToken(),
-        logtoClient.getAccessToken(),
-      ]);
-
-      expect(requester).toHaveBeenCalledWith(tokenEndpoint, expect.anything());
-      expect(accessToken_1).toEqual('access_token_value');
-      expect(accessToken_2).toEqual('access_token_value');
```

Comment:
Why do we remove these accessToken related test cases? Are they covered in `@logto/client`?

Reply:
The answer is YES

Reply:
Besides, in this refactor, all test cases are remained.